src/makeRobot.js

```diff
@@ -38,7 +38,64 @@
  * @return {object}
  */
 function makeRobot(name, wheels, version) {
-  // write code here
+  const robot = {
+    name,
+    wheels,
+    version,
+    coords: {
+      x: 0,
+      y: 0,
+    },
+
+    get info() {
+      return `name: ${this.name},\
+ chip version: ${this.version},\
+ wheels: ${this.wheels}`;
```

Comment:
так переносить не вариант
разбивай не несколько строк
```suggestion
      return `name: ${this.name}, chip version: ${this.version},`
        + `wheels: ${this.wheels}`;
```